### ISSUE TITLE
Doesn't work after update to elementum v:0.0.67

### ISSUE THREAD

**Issue by dancaer69** [elgatito/context.elementum#2]
The "Elementum Play" stopped worked after updated elementum to 0.0.67 version. It worked fine with previous version, but now doesn't play any file. The torrents list dialog opens, but when select any file it doesn't play. In kodi.log I get this error every time I do this action:

```
NOTICE: [plugin.video.elementum] [36mDEBU  torrents     ▶ AddToTorrentsMap  [0mSaving torrent entry for TMDB: "7253902802760462776"
09:47:22.954 T:140277457090304  NOTICE: [plugin.video.elementum] [36mDEBU  database     ▶ AddTorrentHistory  [0mSaving torrent entry for TMDB 7253902802760462776 with infohash 68d787006985b014c323ea1ac96559903adc5e47
09:47:22.958 T:140277457090304  NOTICE: [plugin.video.elementum] [GIN] 2018/12/18 - 07:47:22 | 200 | 37.738836315s |       127.0.0.1 | GET      /search?q=links
09:47:23.407 T:140275544471296   ERROR: [plugin.video.elementum] (elementum.rpc) In Handler method ElementumRPCServer.Player_Open_With_Timeout(u'plugin://plugin.video.elementum/pla...)
09:47:23.409 T:140275544471296   ERROR: [plugin.video.elementum] Unhandled error: TypeError: Player_Open() takes exactly 2 arguments (3 given)
```

I tried it from elementum and from yoda addon. The result and errors are the same.

**Comment by elgatito** [comment]
Make sure you have Elementum 0.0.67 and your system was restarted after update. I can't reproduce this issue with the latest version.

**Comment by elgatito** [comment]
I have found it. Will be fixed in elementum 0.0.68
